demo/server/main.js

```diff
@@ -6,8 +6,16 @@ Meteor.startup(() => {
 });
 
 // setup ProseMeteor on server
-let proseMeteorServer = new ProseMeteorServer({
+const proseMeteorServer = new ProseMeteorServer({
   snapshotIntervalMs: 5000,
   protocol: 'http',
-  docActivityTimeoutMs: (1000 * 60) * 5
- });
+  docActivityTimeoutMs: (1000 * 60) * 5,
+  authentication: {
```

Comment:
for the demo app, I've just added a `canUserEditDoc` hook that returns `false` if the docId is the third doc for an example of having both authenticated an unauthenticated docs on the same page